test/test_helper.rb

```diff
@@ -0,0 +1,240 @@
+require 'bundler/setup'
+
+require 'test/unit'
+
+require 'money'
+require 'yaml'
+require 'json'
+require 'nokogiri'
+
+require 'active_support/core_ext/integer/time'
+require 'active_support/core_ext/numeric/time'
+require 'active_support/core_ext/time/acts_like'
+
+require 'action_controller'
+require 'action_view'
+require 'active_support/core_ext/module/deprecation'
+require 'action_dispatch/testing/test_process'
+
+require 'offsite_payments'
+require 'offsite_payments_latipay'
+
+OffsitePayments.mode = :test
+
+module OffsitePayments
+  module Assertions
```

Comment:
This looks like we've built our own testing framework??? What's going on here?

Reply:
This comes from the parent gems testing framework https://github.com/activemerchant/offsite_payments/blob/master/test/test_helper.rb
A previous comment I made to the PR mentioned that I dedcided to stick with this instead of switching to rspec so that we are more in line with the parent gem